azext_iot/iothub/common.py

```diff
@@ -11,6 +11,11 @@
 from enum import Enum
 
 
+SYSTEM_ASSIGNED_IDENTITY = '[system]'
+
+# Message Endpoint Messages
+INVALID_CLI_CORE_FOR_COSMOS = "This version of the azure cli core does not support Cosmos Db Endpoints for IoT Hub."
```

Comment:
Totally not something to solve in this PR, but just a thought that we have enough messages / constants spread across various `shared.py` and `common.py` - I'm wondering if it would be beneficial to separate (specifically, error message strings or other output formatting) into a `strings.py` module at some point.

Reply:
probably. This will have to be a future discussion that involves clean up of our files.